### PR TITLE
Morty manifest restructure

### PR DESCRIPTION
**Pull request by thenor**
Align manifest restructure to upstream.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -38,24 +39,38 @@ def buildManifest = {String manifest, String bitbake_image ->
...
     stage("Setup bitbake and do fetchall ${bitbake_image}") {
-        // Extract the BSP part of the manifest file name.
-        String bsp = manifest.split("-")[1].tokenize(".")[0]
         // Setup bitbake environment and trigger a 'fetchall'
-        sh "vagrant ssh -c \"TEMPLATECONF=${yoctoDir}/sources/meta-pelux-bsp-${bsp}/conf /vagrant/vagrant-cookbook/yocto/fetch-sources-for-recipes.sh ${yoctoDir} ${bitbake_image}\""
+        confdir = "conf"
+        if (qtauto) {
+            confdir += "-qt"
+        }
+        templateconf="${yoctoDir}/sources/meta-pelux/meta-${bsp}-extras/${confdir} "
```
Comment (tobsan) [inline Jenkinsfile]: This is pre-variants code, so should be updated to reflect the variant handling. There are commits for this in master/pyro.

File: Jenkinsfile [hunk 2]
```diff
@@ -38,24 +39,38 @@ def buildManifest = {String manifest, String bitbake_image ->
...
     stage("Bitbake ${bitbake_image}") {
         sh "vagrant ssh -c \"/vagrant/vagrant-cookbook/yocto/build-images.sh ${yoctoDir} ${bitbake_image}\""
     }
 
+    stage("Copy downloads and cache ${bitbake_image}") {
+        // Archive the downloads and sstate when the environment variable was set to true
+        // by the Jenkins job.
+        if (env.ARCHIVE_CACHE) {
+            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/downloads/ /vagrant/archive/\""
+            sh "vagrant ssh -c \"cp -a ${yoctoDir}/build/sstate-cache/ /vagrant/archive/\""
```
Comment (tobsan) [inline Jenkinsfile]: This has been changed to use rsync instead in pyro/master, should be reflected here as well.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
-
-* Manifest, refers to what `<manifest-name>.xml` file you want to use, for example `pelux-intel.xml`. Each hardware platform targeted by the PELUX reference has its own manifest describing what other git repositories are needed for the build.
-* Image, refers to what version of PELUX that should be built. Currently there are two versions: `core-image-pelux` and `core-image-pelux-qtauto`. The latter being a version that includes NeptuneUI and QtApplicationManager.
+Available images
+^^^^^^^^^^^^^^^^
+There are two different images available: `core-image-pelux-minimal` and
+`core-image-pelux-qtauto-neptune`. The latter being a version that includes
+NeptuneUI and QtApplicationManager.
 
 ### Using vagrant
```
Comment (tobsan) [inline README.md]: This section has been migrated to the SWF.

**Comment by thenor** [comment]
Found and cherry-picked commits which hopefully solves all the found issues.

**Comment by thenor** [comment]
Update in meta-pelux is needed and is on it's way. Update to this PR as soon as that is pulled into the meta-pelux morty branch.

**Pull request merged**